### PR TITLE
update celebratins module.

### PR DESCRIPTION
**Pull request by Akarshcd**
updated a new celebrations module.

### CODE REVIEW

**Review by Daemon-25 — CHANGES_REQUESTED** [review]

File: src/Components/Dashboards/Celebrations.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    color: #0A0A0A;
+
+}
+.clearButton {
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 20px;
+    /* identical to box height, or 143% */
```
[the remark this answers is not in the document]
Comment (Akarshcd) [reply src/Components/Dashboards/Celebrations.css:23]: done!

File: src/Components/Dashboards/Celebrations.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
+import React, { useState } from 'react'
+import { useEffect } from 'react';
+import axios from 'axios';
+import {  GET_PAGE_EMPLOYEES, HOST} from '../Constants/Constants';
+// import celebrationActive from "../../Images/Celebrations_Active.svg";
```
[the remark this answers is not in the document]
Comment (Akarshcd) [reply src/Components/Dashboards/Celebrations.js:5]: done!

File: src/Components/Dashboards/Celebrations.js [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+                     })
+                     else if(new Date().getMonth()===1)
+                     {
+                        upcomingevents.push({
+                          name: "Concept Dash",
+                          msg :"is celebrating  anniversary in "+(28-new Date().getDate())+" days",
+                          pic: cddefault
+                        })  
+                     }
+                   console.log(upcomingevents)
```
[the remark this answers is not in the document]
Comment (Akarshcd) [reply src/Components/Dashboards/Celebrations.js:125]: done!

File: src/Components/Dashboards/Home.js [hunk 4]
```diff
@@ -26,11 +26,67 @@ import { gapi } from "gapi-script";
...
+      marginBottom: "10px",
+      fontFamily: "'Roboto'",
+      fontStyle: "normal",
+      fontWeight: 500,
+      fontSize: "18px",
+      lineHeight: "28px",
+      color: "#0A0A0A",
+    },
+    celebrationsCross: {
+      // marginLeft: '113.6px',
```
[the remark this answers is not in the document]
Comment (Akarshcd) [reply src/Components/Dashboards/Home.js:73]: done!

File: src/Components/Dashboards/Home.js [hunk 5]
```diff
@@ -26,11 +26,67 @@ import { gapi } from "gapi-script";
...
+      marginTop: "16px",
+      background:ishovered?"linear-gradient(134deg, #FAD3E1 0%, rgba(244, 231, 220, 0.89) 23.11%, rgba(240, 244, 216, 0.81) 39.69%, rgba(245, 214, 154, 0.29) 78.22%, rgba(178, 231, 243, 0.00) 100%)":"linear-gradient(134deg, #EFE2F6 0%, rgba(216, 236, 244, 0.81) 39.69%, rgba(239, 226, 246, 0.00) 100%)",
+     filter: "drop-shadow(0px 4px 25px rgba(0, 0, 0, 0.08))",
+      borderRadius: "12px",
+      "&:hover":{
+        background: "black"
+      },
+    },
+    notifModal: {
+      position: "absolute",
```
[the remark this answers is not in the document]
Comment (Akarshcd) [reply src/Components/Dashboards/Home.js:50]: have checked, its working fine!

File: src/Components/Dashboards/Celebrations.js [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+                  let empdata={}      
+                  empdata=res.data.res.map((each)=>{
+                         let ele={}
+                        ele.name=each.First_Name;
+                        ele.Birthday=each.Birthday ? new Date(each.Birthday) : null
+                        ele.JoiningDate=each.Joining_Date ? new Date(each.Joining_Date) : null
+                         return ele
+                  })
+                  setData(empdata)
+                  console.log(empdata)
```
[the remark this answers is not in the document]
Comment (Akarshcd) [reply src/Components/Dashboards/Celebrations.js:60]: done!

**Review by Daemon-25 — APPROVED** [review]

**Pull request merged**